.clj-kondo/config.edn

```diff
@@ -56,6 +56,7 @@
   ;;
   ;; globally-enabled custom linters
   ;;
+  :metabase/check-me-humanize                          {:level :warning}
```

Comment:
`(me/humanize (mc/validate ...))` doesn't work, I wrote a new linter that tells you you're supposed to use `(me/humanize (mc/explain ...))` instead.